tpm2tools/pcr.go

```diff
@@ -77,6 +157,16 @@ func PCRSelection(pcrs *proto.Pcrs) tpm2.PCRSelection {
 	return sel
 }
 
+// FullPcrSel will return a full pcr selection (24 pcrs) with the given
+// hash algo.
+func FullPcrSel(hash tpm2.Algorithm) tpm2.PCRSelection {
+	sel := tpm2.PCRSelection{Hash: hash}
+	for i := 0; i < 24; i++ {
```

Comment:
Use GetPCRCount() to get the PCR count, rather than baking in 24.

Yes, go-tpm bakes this in everywhere. But we shouldn't make this problem worse.

Reply:
done, now a ReadWriter will be passed into this function.